extensions/vertx-http/deployment/src/main/resources/dev-templates/main.html

```diff
@@ -1,77 +1,254 @@
+                                <a id="stopStartButton" class="nav-link"><i class="fas fa-stop"></i></a>
+                            </li>
+                            <li class="nav-item" data-toggle="tooltip" data-placement="bottom" title="Clear log">
+                                <a id="clearLogButton" class="nav-link"><i class="fas fa-trash-alt"></i></a>
+                            </li>
+                            <li class="nav-item" data-toggle="tooltip" data-placement="bottom" title="Zoom out">
+                                <a id="zoomOutButton" class="nav-link"><i class="fas fa-search-minus"></i></a>
+                            </li>
+                            <li class="nav-item" data-toggle="tooltip" data-placement="bottom" title="Zoom in">
+                                <a id="zoomInButton" class="nav-link"><i class="fas fa-search-plus"></i></a>
+                            </li>
+                            <li class="nav-item" data-toggle="tooltip" data-placement="bottom" title="Log levels">
+                                <a id="logLevelsButton" class="nav-link" data-toggle="modal" data-target="#logLevelsModal"><i class="fas fa-layer-group"></i></a>
+                            </li>
+                            <li class="nav-item" data-toggle="tooltip" data-placement="bottom" title="Columns">
+                                <a id="columnsButton" class="nav-link" data-toggle="modal" data-target="#columnsModal"><i class="fas fa-columns"></i></a>
+                            </li>
+                            <li id="filterMenuButton" class="nav-item" data-toggle="tooltip" data-placement="bottom" title="Filter">
+                                <a id="filterButton" class="nav-link" data-toggle="modal" data-target="#filterModal"><i class="fas fa-filter"></i></a>
+                            </li>
+                            <li id="currentFilter" class="navbar-text">
+                            </li>
+                            <li id="resizeButton" class="nav-item"><a class="nav-link"><i class="fas fa-arrows-alt-v"></i></a></li>
+                            <li id="hideLogButton" class="nav-item" data-toggle="tooltip" data-placement="left" title="Hide Log"><a class="nav-link" onclick="hideLog();"><i class="fas fa-caret-down fa-lg"></i></a></li>
+                        </ul>
+                    </div>
+                </nav>
+
+                <div id="loggerManagerLog">
+                    <a id="informationSection"></a>
+                    <a id="followLogButton" data-toggle="tooltip" data-placement="left" title="Follow log (auto scroll)"><i id="followLogIcon" class="text-success fas fa-circle-notch fa-spin"></i></a>
+                    <div id="logTerminal" class="bg-dark text-light ">
+                        <samp id="segmentLog" class="text-wrap">
+                        </samp>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+
+
+        <!-- Filter -->
```

Comment:
Is there any chance we can put the log stuff in a separate file included here somehow? It'd probably make maintenance easier, no?

Reply:
I am sure we should be able to do this with Qute right ?

Reply:
Done. Qute allows for that.